dt-metrics/metrics.php

```diff
@@ -36,11 +36,10 @@ public function __construct(){
             add_action( 'after_setup_theme', function (){
                 $modules = dt_get_option( 'dt_post_type_modules' );
 
-                if ( dt_has_permissions( [ 'multiplier' ] ) ) {
-                    /* Record Types */
-                    $genmap_base_slug = 'personal';
-                    $genmap_base_title = __( 'Personal', 'disciple_tools' );
-                    require_once( get_template_directory() . '/dt-metrics/records/genmap.php' );
+                require_once( get_template_directory() . '/dt-metrics/records/genmap.php' );
+
+                if ( dt_has_permissions( [ 'multiplier' ] ) || dt_has_permissions( [ 'dt_all_access_contacts', 'view_project_metrics' ] ) ) {
```

Comment:
@kodinkat there is no 'multiplier' permission.
multiplier is a role.
I don't think a permission check here for "personal" metrics is needed.